### DISCUSSION TITLE
Compiling error

### DISCUSSION THREAD

**Discussion by Team-Deepflights** [grblHAL/iMXRT1062#40] [General]
Hi,
I have downloaded the latest git over Github and tried to compile it, i got:
`\src\driver.c:1494:51: error: 'driver_cap_t {aka union <anonymous>}' has no member named 'spindle_at_speed'
         if((hal.spindle.get_data = (hal.driver_cap.spindle_at_speed = settings->spindle.ppr > 0) ? spindleGetData : NULL) &&`

The other version I had compiled without trouble, but i have no backup of it ;-)

**Comment by terjeio** [comment]
You have to [update the submodules](https://github.com/grblHAL/core/wiki/Compiling-GrblHAL#using-git-command-line) when downloading a new version.
